DevEdu.API/DevEdu.API/Controllers/User_GroupController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DevEdu.API.Models.InputModels;
+using DevEdu.DAL.Repositories;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DevEdu.API.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class User_GroupController : Controller
+    {
+        private User_GroupRepository _repository;
+        public User_GroupController(User_GroupRepository repository)
+        {
+            _repository = repository;
+        }
+
+        //  api/user_group/group/1/user/2/role/1
+        [HttpPost("group/{groupId}/user/{userId}/role/{roleId}")]

```

Comment:
after moving to GroupController please change routes to "{groupId}/user/{userId}/role/{roleId}"